src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -27,6 +32,7 @@ object SeaCreatureTracker {
 
     private val tracker = SkyHanniTracker("Sea Creature Tracker", { Data() }, { it.fishing.seaCreatureTracker })
     { drawDisplay(it) }
+    private val armorNames = mutableListOf<String>()
```

Comment:
This could also maybe be trophyArmorNames

Reply:
*should*